### PR TITLE
Local Bebb Protocol

### PR DESCRIPTION
**Pull request by patnorris**
Integrates Bebb Protocol such that it can be used locally

To test:
Follow guide in NotesOnLocalDev under section Running Bebb Protocol locally to set up OIM and Bebb Protocol canisters locally

With local UI, test creating a new room
Create Neighbors for the new room
Make sure Neighbors are loaded for the room
Delete Neighbors for the room
Make sure deletions are persisted

### CODE REVIEW

File: NotesOnLocalDev.md [hunk 1]
```diff
@@ -51,3 +51,14 @@ dfx deploy --argument "(
...
 )"
 
+Running Bebb Protocol locally:
+This project's dfx.json specifies Bebb Protocol as a remote canister (i.e. can be used by project's canisters locally but isn't built as it relies on the remote canister being deployed to the local replica from another project). It's newwave. The files for the integration (wasm, did, js, ts) need to be copied over to the integrations/BebbProtocol folder manually if changed.
+Clone Bebb Protocol's repo: https://github.com/patnorris/NewWavePre/tree/firstMotokoVersion
+And use the latest version (e.g. 0.0.3 as of 2023-03)
+Start a local replica in this project
+Deploy this project's canisters
+Go to the folder with Bebb Protocol's latest version and run dfx deploy there (this creates the local Bebb Protocol canister on the local replica started earlier which is thus the same replica this project's canisters are running on)
+Make sure the local canister id for the deployed Bebb Protocol canister is r7inp-6aaaa-aaaaa-aaabq-cai (required by this project to make the calls on the local replica)
```
Comment (tiusty) [inline NotesOnLocalDev.md:61]: My canister id is different

Comment (tiusty) [reply NotesOnLocalDev.md:61]: Should it be the same? I cloned it, went to the latest branch and run dfx deploy in the 0.0.3 folderr

Comment (patnorris) [reply NotesOnLocalDev.md:61]: As far as I know, the local replica always assigns the same ids to the canisters being created in order (i.e. id1 to first canister, id2 to 2nd, etc), so it should be the same. I'll take another look into this too

**Comment by patnorris** [comment]
The latest commit updates the canister id for the local Bebb Protocol to make the setup reproduceable and adds some notes about the setup.

**Comment by tiusty** [comment]
I tested this again with the instructions you put and couldn't get it to work. We should talk it over to make sure I am not missing anything at some point

**Comment by tiusty** [comment]
Got it to work!

**Review by tiusty — APPROVED** [review]

**Pull request merged**